<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSpesSymb</class>
 <widget class="QDialog" name="DialogSpesSymb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вставка дополнительных символов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_0.ico</normaloff>:/spec_0.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_1.ico</normaloff>:/spec_1.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_2.ico</normaloff>:/spec_2.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_3.ico</normaloff>:/spec_3.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_4.ico</normaloff>:/spec_4.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_5.ico</normaloff>:/spec_5.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_6.ico</normaloff>:/spec_6.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_8">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_7.ico</normaloff>:/spec_7.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_9">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_8.ico</normaloff>:/spec_8.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_10">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_9.ico</normaloff>:/spec_9.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_11">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_10.ico</normaloff>:/spec_10.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_12">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_11.ico</normaloff>:/spec_11.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_13">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_12.ico</normaloff>:/spec_12.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_14">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_13.ico</normaloff>:/spec_13.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_15">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/spec_14.ico</normaloff>:/spec_14.ico</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
